mlflow/models/evaluation/base.py

```diff
@@ -1357,15 +1397,22 @@ def evaluate(
                         def fn(model_input):
                             return model.predict(model_input)
 
+        If omitted, it indicates a static dataset will be used for evaluation instead of a model.
+        In this case, the ``data`` argument must be a Pandas DataFrame that contains model outputs,
+        and the ``predictions`` argument must be the name of the column in ``data`` that contains
+        model outputs.
+
     :param data: One of the following:
 
                  - A numpy array or list of evaluation features, excluding labels.
 
-                 - A Pandas DataFrame or Spark DataFrame, containing evaluation features and
-                   labels. If ``feature_names`` argument not specified, all columns are regarded
-                   as feature columns. Otherwise, only column names present in ``feature_names``
-                   are regarded as feature columns. If it is Spark DataFrame, only the first 10000
-                   rows in the Spark DataFrame will be used as evaluation data.
+                 - A Pandas DataFrame or Spark DataFrame, containing evaluation features,
+                   labels, and optionally model outputs. If ``feature_names`` argument not
```

Comment:
Can we indicate when model outputs are required, i.e. when `model` is unspecified?

Reply:
Updated